recipe/build.sh

```diff
@@ -19,15 +26,19 @@ fi
 # value at run-time to get the expected amount of parallelism.
 export OPENBLAS_NUM_THREADS=1
 
-WITH_BLAS_LIB="-L${PREFIX}/lib -lblas"
-WITH_LAPACK_LIB="-L${PREFIX}/lib -llapack"
-
 ./configure \
-    --prefix="${PREFIX}" \
-    --exec-prefix="${PREFIX}" \
-    --with-blas-lib="${WITH_BLAS_LIB}" \
-    --with-lapack-lib="${WITH_LAPACK_LIB}" \
-    --enable-gnu-packages
+  --prefix="${USE_PREFIX}" \
+  --exec-prefix="${USE_PREFIX}" \
+  --with-blas-lib="${BLAS_LIB}" \
+  --with-lapack-lib="${LAPACK_LIB}" \
+  --disable-pkg-config \
+  ${EXTRA_FLAGS} || cat Clp/config.log
```

Comment:
Maybe try to print the config.log always for now.
I saw some checks on your configure file that will test the user defined libs first. Maybe sth goes wrong there.